clientconn.go

```diff
@@ -266,7 +267,7 @@ func DialContext(ctx context.Context, target string, opts ...DialOption) (conn *
 	// Configure idleness support with configured idle timeout or default idle
 	// timeout duration. Idleness can be explicitly disabled by the user, by
 	// setting the dial option to 0.
-	cc.idlenessMgr = newIdlenessManager(cc, cc.dopts.idleTimeout)
+	cc.idlenessMgr = idle.NewManager(idle.ManagerOptions{Enforcer: (*idler)(cc), Timeout: cc.dopts.idleTimeout, Logger: logger})
```

Comment:
Just curious why you find this single line literal struct initialization more readable than the more common multiline one.

Reply:
IDK personally I find the parameters passed in here are really obvious and uninteresting, and I'd rather have one operation per line more (for denser code, which is easier to scroll through) than I'd like to be able to more spaciously see all the details of the values.  If I needed to do tricky math or call a function to set them, I'd probably make it multi-line.

Same reason I dislike most `var` blocks, particularly in functions, e.g. 

```go
var (
	i int
	p peer.Peer
)
```

Those variables aren't even related; why group them when it takes even more lines than declaring them separately?